VMEncryption/main/handle.py

```diff
@@ -751,7 +751,11 @@ def enable():
         encryption_config = EncryptionConfig(encryption_environment=encryption_environment, logger=logger)
         if encryption_config.config_file_exists():
             existing_volume_type = encryption_config.get_volume_type()
-
+        #log to capture lsblk before encryption view.
+        disk_util.log_lsblk_output()
+        if public_settings.get(CommonVariables.EncryptionEncryptionOperationKey) == CommonVariables.EnableEncryptionFormatAll:
+            #in case of stop start unmount /mnt to avoid resource disk encryption issues.
+            disk_util.umount('/mnt')
```

Comment:
where will the resource disk get mounted again?

Have you considered creating a separate mount point for BEK which is not under /mnt? For example, /azure_bek_disk ?